### DISCUSSION TITLE
Search Songs Queue

### DISCUSSION THREAD

**Discussion by shiinayane** [ciderapp/Cider#1462] [Feature Request]
When you search sth. and want to play all the results, you click on See All and randomly play a song.
You'll find that only this song you clicked is added into the queue.
This bug also happens in artist page, when you enter Top Songs - See All.
See the gifs below.
![2022-09-23 13-27-21](https://user-images.githubusercontent.com/53814845/191897473-5630f526-b602-48a4-92ea-fe9bedbd8db8.gif)
![2022-09-23 13-27-43](https://user-images.githubusercontent.com/53814845/191897478-6f784732-b815-421c-99c1-047f24f48206.gif)

**Comment by coredev-uk** [comment]
The see all view is only search results still, that's not how it operates.

**Comment by shiinayane** [comment]
Well, I see, still hope this can be improved in the future.

**Comment by coredev-uk** [comment]
I will convert this to a discussion as its a feature request.
